### ISSUE TITLE
Presentation schedule for Pelikan Cache

### ISSUE THREAD

**Issue by danielamiao** [cncf/tag-storage#79]
Hi tag-storage people 👋

I have been working with a team of ex-Twitter engineers on https://pelikan.io/ (which is now a fork of https://github.com/twitter/pelikan). We'd like to present at the next TAG storage meeting, and get some feedback on the project. How should we go about signing up? 

Specific goals:
- Get feedback on the project in general. How does it fit in the current CNCF Storage Landscape?
- Get feedback on types of talks that may be interesting for the KubeCon/CNCF audience, we are hoping to submit to KubeCon NA 2023.
- Get advice on how to jump start the OSS community, we have plans to reach out to tag contributor strategy as well to get advice here.

Thank you!

**Comment by chira001** [comment]
Hi @danielamiao,

That sounds very interesting.   We meet on the 2nd and 4th Wednesday of the month, and the next call is this Wednesday at 8am Pacific time.  See https://www.cncf.io/calendar/ for details.   Would you be able to make this week's call?

Kind Regards,
Alex
(Co-chair CNCF Storage TAG)

cc @xing-yang

**Comment by danielamiao** [comment]
@chira001 Yes! I can make it to this week's call and chat about the project. Is there an agenda where I can sign up to hold a slot? Or this issue is enough?

**Comment by chira001** [comment]
Brilliant - thank you @danielamiao .   We can add you to the agenda, and I'll send a note out to the mailing list too.    Is there a quick summary of the project that want me to include, or are the links in the original post the most appropriate?

**Comment by danielamiao** [comment]
@chira001 Thank you! The links in the original post are most appropriate, I'll put together a summary for discussion tomorrow as well.

**Comment by chira001** [comment]
Great.   The call is at 8am Pacific
Conf info:   https://zoom.us/j/2920471159?pwd=em1JbE44MktjZE4vbnJtUUFQcGZwdz09 (Passcode: 77777)

**Comment by xing-yang** [comment]
Hi @danielamiao,

When you gave a presentation to TAG Storage back in March, we recommended that you donate the Pelikan project to CNCF as it is a great fit and it can help you grow the community.  Just want to check to see how you are doing and whether you are interested in donating the project to CNCF. 

Thanks!
Xing

cc @chira001 @nikhita
